### DISCUSSION TITLE
<Break>

### DISCUSSION THREAD

**Discussion by daniel-stilman** [Stability-AI/StableSwarmUI#340] [Q&A]
I have scoured the documentation and found nothing about this anywhere: what's the deal? It doesn't seem to do what BREAK, for example, does on A1111, as the token count doesn't change when I use it (well, actually, it seems to rise by one, possibly indicating it is being seen as a token).

How do I use multi-promts?

**Comment by mcmonkey4eva** [comment]
The token counter probably needs to be updated to recognize `<break>`, it does work the same as `BREAK` in actual generations

**Reply by mcmonkey4eva** [reply]
Update: I pushed a commit that will now count properly
